awscli/customizations/sso/utils.py

```diff
@@ -65,6 +65,25 @@ def open_browser_with_original_ld_path(url):
         webbrowser.open_new_tab(url)
 
 
+class PrintOnlyHandler(object):
```

Comment:
We probably don’t need the object inherentence anymore.

Reply:
Or even better, it would be great if we can make a base class that both the `PrintOnlyHandler` and `OpenBrowserHandler` subclass from. Specifically, that will make it clear what the interface that is needed as a `on_pending_authorization` handler.